framework/UPGRADE.md

```diff
@@ -96,7 +90,13 @@ Upgrade from Yii 2.0.37
     code if those parents implement the `useStrictMode` checks.
 
   > Warning: in case `openSession()` and/or `writeSession()` functions do not implement the `useStrictMode` code
-    the session could be stored under the forced id without warning even if `useStrictMode` is enabled.
+    the session could be stored under a malicious id without warning even if `useStrictMode` is enabled.
+
+Upgrade from Yii 2.0.37
```

Comment:
That's this release as well.

Reply:
How do you mean? In my understanding the next version will be 2.0.39, so the upgrade should be from version 2.0.38? (in the original PR it was still filed under the 2.0.37 version)